### PR TITLE
WRN-2465: Date/TimePicker: add prop to hide label

### PR DESCRIPTION
**Pull request by Sangwook-Lee**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [ ] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
The GUI requirement is to display the Date/TimePicker without a label.
There are several scenarios for hiding time/date labels.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
It Added prop `noLabel`  to DatePicker and TimePicker.
And then, if `label` is not valid, not displays Heading.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-2465

### CODE REVIEW

File: DatePicker/DatePickerBase.js [hunk 1]
```diff
@@ -147,6 +147,15 @@ const DatePickerBase = kind({
 		 */
 		monthAriaLabel: PropTypes.string,
 
+		/**
+		 * Hides the label that displays the date.
+		 *
+		 * @type {Boolean}
+		 * @default false
```
Comment (MikyungKim) [inline DatePicker/DatePickerBase.js:154]: Please remove this line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 ### Added
 
 - `sandstone/ImageItem` public classname `imageIcon`
+- `sandstone/DatePicker` and `sandstone/TimePicker` prop `noLabel` to hide label
```
Comment (MikyungKim) [inline CHANGELOG.md:10]: Please put this log at the first.

File: TimePicker/TimePickerBase.js [hunk 3]
```diff
@@ -194,6 +194,15 @@ const TimePickerBase = kind({
 		 */
 		minuteAriaLabel: PropTypes.string,
 
+		/**
+		 * Hides the label that displays the time.
+		 *
+		 * @type {Boolean}
+		 * @default false
```
Comment (MikyungKim) [inline TimePicker/TimePickerBase.js:201]: same here.

**Review by MikyungKim — COMMENTED** [review]
Please add screenshot tests for this prop.

File: DatePicker/DatePickerBase.js [hunk 4]
```diff
@@ -147,6 +147,11 @@ const DatePickerBase = kind({
 		 */
 		monthAriaLabel: PropTypes.string,
 
+		/**
```
Comment (MikyungKim) [inline DatePicker/DatePickerBase.js:150]: My intention was to remove `default: false` not the entire jsdoc ;)

**Review by MikyungKim — APPROVED** [review]
:lgtm:
unit test, ui test, and ss test are passed(# 496).

**Review by 0x64 — APPROVED** [review]

**Pull request merged**